genericjpa/standalone/src/main/java/org/hibernate/search/SearchFactory.java

```diff
@@ -0,0 +1,107 @@
+package org.hibernate.search;
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+
+import java.util.Set;
+
+import org.apache.lucene.analysis.Analyzer;
+
+import org.hibernate.search.indexes.IndexReaderAccessor;
+import org.hibernate.search.metadata.IndexedTypeDescriptor;
+import org.hibernate.search.query.dsl.QueryContextBuilder;
+import org.hibernate.search.stat.Statistics;
+
+/**
+ * Provide application wide operations as well as access to the underlying Lucene resources.
+ *
+ * @author Emmanuel Bernard
+ * @author Hardy Ferentschik
+ * @author Sanne Grinovero
+ */
+public interface SearchFactory {
```

Comment:
ah good catch. In my code i used this in the same way as the one from orm. This has to be put into some common module.